practice_app/frontend/api.js

```diff
@@ -32,7 +32,37 @@ function getTweets() {
     }, {
         "id": "2",
         "user": "alesta",
-        "content": "ne dedin sen?",
+        "content": "Sense child d wisdom wcuten whole as match asked. Pleasure exertion put add entrance distance drawings. In equally matters showing greatly it as. Want name any wise are able park when. Saw vicinity judgment remember finished men throwing.",
+        "rt_count": 3,
+        "like_count": 99
+    }, {
+        "id": "2",
+        "user": "alesta",
+        "content": "alestaSense child do state to defer mr of forty. Become latter but nor abroad wisdom waited.alestaSense child do state to defer mr of forty. Become latter but nor abroad wisdom waited.alestaSense child do state to defer mr of forty. Become latter but nor abroad wisdom waited.123 ",
+        "rt_count": 3,
+        "like_count": 99
+    }, {
+        "id": "2",
+        "user": "alesta",
+        "content": "aaaaaaaaaaaaaaaaaaaaaaaaaaaaaasdas123123 qwe 123 12aaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa",
+        "rt_count": 3,
+        "like_count": 99
+    }, {
+        "id": "2",
+        "user": "alesta",
+        "content": "aaaaaaaaaaaaaaaaaaaaaaaaaaaaaasdas123123 qwe 123 12aaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa",
+        "rt_count": 3,
+        "like_count": 99
+    }, {
+        "id": "2",
+        "user": "alesta",
+        "content": "aaaaaaaaaaaaaaaaaaaaaaaaaaaaaasdas123123 qwe 123 12aaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa",
+        "rt_count": 3,
```

Comment:
We might want to try this out with different field values. Who knows, stuff might break.

Reply:
The field which might be a trouble is the content field. Twitter allows the user to use 280 char. I tried the looking  up to 280 char and arranged the weight and height. Ofcourse we need to try with more realistic test cases.